public/app/plugins/panel/xychart/config.ts

```diff
@@ -27,6 +27,13 @@ export function getScatterFieldConfig(cfg: ScatterFieldConfig): SetFieldConfigOp
       },
     },
 
+    disableStandardOptions: [
+      FieldConfigProperty.Thresholds,
+      FieldConfigProperty.Mappings,
+      FieldConfigProperty.Min,
+      FieldConfigProperty.Max,
+    ],
+
```

Comment:
Are we temporarily disabling these options until we figure out a way for us to handle these options for both x + y axes on a global level? Or do we need to disable these options for panels that allow for x axis configuration and then have special options on a panel by panel basis?

Maybe this topic warrants a design doc ;)